src/app/crud/crud_robot_type.py

```diff
@@ -0,0 +1,49 @@
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.future import select
+from typing import Any, Dict, Optional, Union, List
+
+from app.crud.base import CRUDBase
+from app.models.robot_type import RobotType
+from app.schemas.robot_type import RobotTypeCreate, RobotTypeUpdate
+
+
+class CRUDRobotType(CRUDBase[RobotType, RobotTypeCreate, RobotTypeUpdate]):
+    async def create(
+            self, db: AsyncSession, *, obj_in: RobotTypeCreate
+    ) -> RobotType:
+        db_obj = RobotType(
+            name=obj_in.name,
+            vendor=obj_in.vendor
+        )
+        db.add(db_obj)
+        await db.commit()
+        await db.refresh(db_obj)
+        return db_obj
+
+    async def update(
+            self, db: AsyncSession, *, db_obj: RobotType, obj_in: Union[RobotTypeUpdate, Dict[str, Any]]
+    ) -> RobotType:
+        if isinstance(obj_in, dict):
+            update_data = obj_in
+        else:
+            update_data = obj_in.dict(exclude_unset=True)
```

Comment:
```suggestion
        update_data = obj_in if (obj_in, dict) else obj_in.dict(exclude_unset=True)
```
What do you think about ternary operators?

Reply:
Like them, applied it!